ui/tests/integration/components/icon-test.js

```diff
@@ -20,12 +20,35 @@ module('Integration | Component | icon', function(hooks) {
     await render(hbs`<Icon class="al" aria-label="Testing" />`);
     assert.dom('.al').hasAttribute('aria-label', 'Testing', 'renders aria-label');
 
-    await render(hbs`<Icon @glyph="vault-logo" @size="s"/>`);
+    await render(hbs`<Icon @name="vault-logo" @sizeClass="s"/>`);
     assert.dom('.hs-icon').hasClass('hs-icon-s', 'adds the size class');
 
     let promise = waitForError();
-    render(hbs`<Icon @glyph="vault-logo" @size="no"/>`);
+    render(hbs`<Icon @name="vault-logo" @sizeClass="no"/>`);
     let err = await promise;
-    assert.ok(err.message.includes('The size property of'), "errors when passed a size that's not allowed");
+    assert.ok(
+      err.message.includes('The sizeClass property of'),
+      "errors when passed a sizeClass that's not allowed"
+    );
+  });
+
+  test('it should render FlightIcon', async function(assert) {
+    assert.expect(5);
```

Comment:
What is this?

Reply:
This ensures that an explicit number of assertions are run within the test, in this case 5. It's a force of habit for me but basically just helps to guard against unexpected assertions or unintended modifications to a test perhaps.